pkg/web/templates/partials/sunrise/message_preview.html

```diff
@@ -0,0 +1,182 @@
+<div id="preview-msg-container" class="modal-box mx-auto relative p-4 max-w-5xl">
+  <div class="max-h-[450px] overflow-y-auto">
+    <!-- Modal header -->
+    <div class="md:flex items-center justify-between p-4 border border-black bg-neutral">
+      <div class="mb-4 md:mb-0">
+        <h1 class="text-xl font-bold text-gray-900 lg:text-2xl">
```

Comment:
The modal is designed to appear similar to the transaction preview modal displayed for TRISA and TRP requests.